3.optimizing_CP/run_sam/scripts/run_sam2_microscopy.py

```diff
@@ -0,0 +1,673 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# This notebook solves the tracking issue by using [SAM2](https://github.com/facebookresearch/segment-anything-2/tree/main).
+# Here I use the pretrained model to segment the objects in the video.
```

Comment:
Recommend including link to the model on GitHub.

Reply:
```suggestion
# Here I use the pretrained model to segment the nuclei in the video from the ___ channel.
```

Reply:
Are you planning on using this to segment whole cells as well? This suggestion might be incorrect if so.

Reply:
I believe from my understanding that you are using this for nuclei and then can use that in CellProfiler to detect whole cells using the segmentation parameters in that pipeline. This is just detecting the "primary" objects.

Reply:
Yes, correct, I willl change this documentation here